src/core/manager.c

```diff
@@ -2610,23 +2610,23 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                         uint32_t id;
 
                         if (safe_atou32(val, &id) < 0)
-                                log_debug("Failed to parse current job id value %s", val);
```

Comment:
Well, we expect to remove stuff occasionally, but I don't think we ever mean to make old serialization formats unparseable on purpose. Are you aware of any instances of that happening? I'm aware of "kdbus-fd=" going away, and we explicitly silently ignore that.